lib/v2/bilibili/article.js

```diff
@@ -13,19 +12,41 @@ module.exports = async (ctx) => {
         },
     });
     const data = response.data.data;
-
-    ctx.state.data = {
-        title: `${name} 的 bilibili 专栏`,
-        link: `https://space.bilibili.com/${uid}/#/article`,
-        image: data.articles && data.articles[0].author.face,
-        description: `${name} 的 bilibili 专栏`,
-        item:
-            data.articles &&
-            data.articles.map((item) => ({
+    const title = `${name} 的 bilibili 专栏`;
+    const link = `https://space.bilibili.com/${uid}/#/article`;
+    const description = `${name} 的 bilibili 专栏`;
+    const item = await Promise.all(
+        data.articles.map(async (item) => {
+            const art_url = `https://www.bilibili.com/read/cv${item.id}`;
+            const itemData = await ctx.cache.tryGet(
+                art_url,
+                async () =>
+                    (
+                        await got({
+                            method: 'get',
+                            url: art_url,
+                            headers: {
+                                Referer: `https://space.bilibili.com/${uid}/`,
+                            }
+                        })
+                    ).data
+            );
+            const content = cheerio.load(itemData);
+            const eDescription = content('#read-article-holder').html();
+            const publishDate = new Date(item.publish_time * 1000).toUTCString();
```

Comment:
```suggestion
            const publishDate = parseDate(item.publish_time * 1000);
```
docs: https://docs.rsshub.app/joinus/pub-date.html#shi-yong-gong-ju-lei-parse-date